src/components/Slideover/Slideover.vue

```diff
@@ -53,17 +53,37 @@ export default {
     },
   },
   watch: {
-    isOpen() {
-      this.openBlokUI = this.isOpen
+    isOpen(state) {
+      if (state) {
+        this.handleOpenSlide()
+      } else {
+        this.handleCloseSlide()
+      }
     },
   },
   methods: {
     /**
      * handler for close the component
      */
     handleCloseSlide() {
-      this.openBlokUI = false
-      this.$emit('hide')
+      if (this.openBlokUI) {
+        this.openBlokUI = false
+        this.$nextTick(() => {
+          this.$emit('hide')
+        })
```

Comment:
Same question here, similar to the Modal opening/closing handlers.

Reply:
The same answer.